src/GovUk.Education.ExploreEducationStatistics.Content.Model.Test/ReleaseTests.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using Xunit;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Model.Test
```

Comment:
Can we rename the project and namespace from `Test` to `Tests` this keeps it consistient and also simplies the pipline with runs tests for all project that end in `Tests` or `tests`

Reply:
Done